### DISCUSSION TITLE
NCLC Tag Editing

### DISCUSSION THREAD

**Discussion by Sweetmate** [videovillage/community#136] [Ideas]
Would love for Screen to be able to edit NCLC tags of a quicktime or mp4.

https://forum.blackmagicdesign.com/viewtopic.php?f=21&t=95658

https://github.com/bbc/qtff-parameter-editor

**Comment by gregcotten** [comment]
This is a great idea! I have track editing on the roadmap, but I could probably get basic atom editing done sooner.

**Comment by julesvm** [comment]
Yes to this! Would be huge

**Comment by lukeguy77** [comment]
would really love to see this as a feature. would be so helpful!

**Comment by benrohel** [comment]
+1

**Comment by radambe** [comment]
Just came to submit this same feature request and found it to be already here.  I second this request with passion!
